src/ui/modals/smithing-interface.js

```diff
@@ -0,0 +1,97 @@
+/* import { ModalWindow } from "../modal-window.js";
```

Comment:
I think we should delete the file instead of just commenting stuff out, can look back at git history if we need the code again